.gitignore

```diff
@@ -0,0 +1,3 @@
+go.sum
```

Comment:
The `go.sum` should not be ignored.

Reply:
My bad, I just started learning Go and thought it was something similar to Node's `package-lock.json`. Issue in the following commit:

https://github.com/kudrykv/latex-yearly-planner/pull/27/commits/abe2000686ad75946a3cf085b202573b70644e98